lib/middleware/check-progress.js

```diff
@@ -18,7 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
 
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = controller._getForkTarget(req, res);
```

Comment:
That's what I mentioned the other day, I did this to show this method shouldn't be overwritten in form-controller. If you prefer I can name it without the underscore, but then in form-controller it would look like any other method than can be overwritten.

Unless we add a `getForkTarget` method in form controller which simply calls this method and returns the result? If we did this though, it would look as though `getForkTarget` can be overwritten in child controllers, but that would break this middleware.

What we want is a publicly accessible method, that cannot be overwritten in child classes. Ideas @easternbloc @JoeChapman @gavboulton @HughePaul ?